### PR TITLE
Final group capstone - Book an Appointment -Final PR

### PR DESCRIPTION
**Pull request by dodoburner**
# Completed Project Requirements
https://github.com/EyasuTeshome/final-capstone-project-front-end/pull/22

## Setup

Run rails db:seed to populate cars and an admin user in database

Login as admin with the following credentials

Email: admin@gmail.com
password: 111111

### Unexpected issues
- Members in team were 5 when we created the Kanban board. So we worked on the requirements for 5 members. But one member left, whiles we were almost at the end of the project. Some of the optional requirements for the 3-4 members were left out.

https://github.com/learnwithalfred/backend-microverse-final-capstone/issues/9

## Technical set up

 Project was setup using two repositories

- Front-end https://github.com/EyasuTeshome/final-capstone-project-front-end
- Backend https://github.com/learnwithalfred/backend-microverse-final-capstone

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,156 +64,116 @@ After you're finished please remove all the comments and instructions!
...
 ### Usage
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
 ```sh
-  rails server
+  rails s
```
Comment (Meltrust) [inline README.md]: - [ ] Kindly, please add the required db commands that your users will need to setup the db to run the project

```
rails db:create
rails db:migrate
rails db:seed
```

That way, your users will run your project smoothly 💪

File: README.md [hunk 2]
```diff
@@ -94,156 +64,116 @@ After you're finished please remove all the comments and instructions!
...
 ### Run tests
 
 To run tests, run the following command:
 
-<!--
-Example command:
 
 ```sh
-  bin/rails test test/models/article_test.rb
+  rspec
```
Comment (Meltrust) [inline README.md:130]: - [x]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

- [x] 1 test shows up as pending (not implemented):

image

image


Kindly delete that file `reservations_spec.rb` if not using it. 👍

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @dodoburner,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:

-  Tests are passing✔️
-  API is working well✔️ 
-  Good readme ✔️ 



#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._

_I strongly recommend you take them into account as they can improve your final evaluation._





Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

File: README.md (hunk 2 above)
Comment (learnwithalfred) [reply README.md:130]: Resolved

**Review by Meltrust — APPROVED** [review]
#### Hi @dodoburner,


# **Wow, you did it 🎉**
Brilliant
## Thank you for the changes implemented 💪 🥇 ㊗️ 
#### _Unless you want to add more features, go ahead to your final presentation ⏩ ⏩ ⏩_ 
## You are about to finish the Microverse program.  You have come a long way!!!




### Good luck in the software industry!!  I'll see you there. ✨
### **Congratulations!!!!!!**  🎉
applause

### To highlight

- The API is working well✔️ 
- Nice readme✔️ 
- Nice models and controllers✔️ 
- Good code implementation ✔️ 



Cheers and Happy coding!👏👏👏


------

**Pull request merged**